Runtimes/Supplemental/Synchronization/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.29)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE YES) # ToDo: What does this do?
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake/modules")
+
+if($ENV{BUILD_NUMBER})
+  math(EXPR BUILD_NUMBER "$ENV{BUILD_NUMBER} % 65535")
+  set(BUILD_NUMBER ".${BUILD_NUMBER}")
+endif()
+project(SwiftSynchronization
+  LANGUAGES Swift
+  VERSION 6.1.0${BUILD_NUMBER})
+
+include(GNUInstallDirs)
+
+if(NOT PROJECT_IS_TOP_LEVEL)
+  message(SEND_ERROR "Swift Synchronization must build as a standalone project")
+endif()
+
+set(${PROJECT_NAME}_SWIFTC_SOURCE_DIR
+  "${PROJECT_SOURCE_DIR}/../../../"
+  CACHE FILEPATH "Path to the root source directory of the Swift compiler")
```

Comment:
Do you reference anything from there? I think that we should just drop this if possible. The supplemental libraries don't tend to reach into the sources of the core libraries AFAIK.

Reply:
\*gyb enters chat\*

Reply:
Though, for the purposes of the supplemental libraries, the gyb module should probably reach up into the compiler bits so that each of the supplemental projects don't need to explicitly.